### PR TITLE
better dependency declaration

### PR DESCRIPTION
**Pull request by dgomezg**
To address Issue #1

### CODE REVIEW

File: microservices/rental-command/pom.xml [hunk 1]
```diff
@@ -3,24 +3,23 @@
...
-            <groupId>${project.groupId}</groupId>
-            <artifactId>core-api</artifactId>
-            <version>${project.version}</version>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
```
Comment (CodeDrivenMitch) [inline microservices/rental-command/pom.xml:22]: can´t we move actuator to the microservices pom?

File: requests.http [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ### Generate bikes
```
Comment (CodeDrivenMitch) [inline requests.http:1]: Maybe it's better to leave the values as they were?

**Review by CodeDrivenMitch — APPROVED** [review]
Two small things, but no big concerns. Feel free to change them and then merge them

File: requests.http (hunk 2 above)
Comment (dgomezg) [reply requests.http:1]: Yeap, will do, this is a local change I did to run locally and it leaked to the commit...

**Comment by dgomezg** [comment]
Resolves #2

**Pull request merged**